pubsublite/admin.go

```diff
@@ -147,9 +147,41 @@ func (ac *AdminClient) Topics(ctx context.Context, parent string) *TopicIterator
 	}
 }
 
+// OffsetLocation refers to the location of an offset with respect to the
+// message backlog.
+type OffsetLocation int
+
+const (
+	// UnspecifiedOffsetLocation represents an OffsetLocation that has not
+	// been set.
+	UnspecifiedOffsetLocation OffsetLocation = iota
+
+	// End refers to the offset past all currently published messages. End
+	// skips the entire message backlog.
+	End
+
+	// Beginning represents the offset of the oldest retained message.
+	Beginning
+)
+
+// CreateSubscriptionOpts specify the options to use when creating a
+// subscription.
+type CreateSubscriptionOpts struct {
```

Comment:
Nit: Perhaps CreateSubscriptionOptions? Just to match CreateSubscriptionWithOptions.

Reply:
Done.  Now using a CreateSubscriptionOption.